tinyman/v1/staking/__init__.py

```diff
@@ -40,13 +40,13 @@ def prepare_update_transaction(app_id: int, sender, suggested_params):
     return TransactionGroup([txn])
 
 
-def prepare_commit_transaction(app_id: int, program_id: int, program_account: str, pool_asset_id: int, amount: int, reward_asset_id: int, sender, suggested_params):
+def prepare_commit_transaction(app_id: int, program_id: int, program_account: str, pool_asset_id: int, amount: int, sender, suggested_params):
     txn = ApplicationNoOpTxn(
         index=app_id,
         sender=sender,
         sp=suggested_params,
         app_args=['commit', int_to_bytes(amount)],
-        foreign_assets=[pool_asset_id, reward_asset_id],
+        foreign_assets=[pool_asset_id],
         accounts=[program_account],
         note=b'tinymanStaking/v1:b' + int_to_bytes(program_id) + int_to_bytes(pool_asset_id) + int_to_bytes(amount)
```

Comment:
Kept the note for now.
https://hipo.slack.com/archives/C0394J0P7DL/p1650903040169479?thread_ts=1650895723.529299&cid=C0394J0P7DL